code-studio/build-js.js

```diff
@@ -17,10 +21,11 @@ commander
     .parse(process.argv);
 
 // Run build (exits on failure)
+// TODO - think about how watchify will work for multiple commands
```

Comment:
Right now, watch mode would watch code-studio, but not initApp (because that command doesn't finish). Solution is probably to have these run in parallel instead of sequence. Will do that at a future time.